testworld/v2_documents_test.go

```diff
@@ -51,3 +51,28 @@ func TestV2DocumentCreate_next_version(t *testing.T) {
 	// alice should not have this version
 	nonExistingDocumentVersionCheck(alice.httpExpect, alice.id.String(), "documents", params)
```

Comment:
can you adapt commit to the functions NewDocument and Exisitng document?
We already have created a pending document at this point instead of re-creating a new one ?

Reply:
I rather keep it separate since it is a separate API call independent of the other tests. Easier to track.

Reply:
the new test is making the same API calls as the new one. Adapting to old tests ensures we do not have a lot of tests doing the same. WDYT ?